### PR TITLE
Update admission controllers topic

### PR DESCRIPTION
**Pull request by traci-morrison**
### Proposed changes

- https://docs.docker.com/ee/ucp/admin/configure/admission-controllers/
- Updated Note format
- Added "Where to go next" section

### CODE REVIEW

**Comment by traci-morrison** [comment]
@Dawn-Wood @KoryKessel-Docker PTAL

File: ee/ucp/admin/configure/admission-controllers.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-description: Learn about how admission controllers are used in docker.
+description: Learn about how admission controllers are used in Docker.
 keywords: cluster, psp, security
 ---
 
 
 >{% include enterprise_label_shortform.md %}
 
-This is the current list of admission controllers used by Docker:
+Admission controllers are plugins that govern and enforce how the cluster is used. There are two types of admission controllers used, Default and Custom.
```
Comment (KoryKessel-Docker) [inline ee/ucp/admin/configure/admission-controllers.md]: Make *Default* and *Custom* bold.

File: ee/ucp/admin/configure/admission-controllers.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+
+This topic describes the current list of admission controllers used by Docker.
```
Comment (KoryKessel-Docker) [inline ee/ucp/admin/configure/admission-controllers.md]: In my opinion, this line is unnecessary.

File: ee/ucp/admin/configure/admission-controllers.md [hunk 3]
```diff
@@ -22,7 +24,7 @@ This is the current list of admission controllers used by Docker:
 - [PodSecurityPolicy](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy)
 
 ### Custom
-- **UCPAuthorization**
+- **UCPAuthorization:**
 This custom admission controller does several things:
```
Comment (KoryKessel-Docker) [inline ee/ucp/admin/configure/admission-controllers.md]: In my opinion, this lead-in is unnecessary.

File: ee/ucp/admin/configure/admission-controllers.md [hunk 4]
```diff
@@ -52,6 +54,11 @@ not run on swarm-only nodes, which UCP taints with
 affinity to prevent pods from running on manager nodes depending on UCP's
 settings.
 
-**Note:** you cannot enable or disable your own admission controllers. For more information, see [Supportability of custom kubernetes flags in universal control plane](https://success.docker.com/article/supportability-of-custom-kubernetes-flags-in-universal-control-plane)
+> **Note** 
+> 
+> You cannot enable or disable your own admission controllers. For more information, see [Supportability of custom Kubernetes flags in universal control plane](https://success.docker.com/article/supportability-of-custom-kubernetes-flags-in-universal-control-plane).
+
```
Comment (KoryKessel-Docker) [inline ee/ucp/admin/configure/admission-controllers.md]: Does this note apply only to the Custom admission controllers (that is how "your own" reads to me) or to all admission controllers?

If the note only applies to Custom admission controllers, I suggest editing the first sentence to read, "Custom admission controllers cannot be enabled or disabled by the user."

If the note applies to all admission controllers, I suggest editing the first sentence to read, "Admission controllers cannot be enabled or disabled by the user."

**Comment by traci-morrison** [comment]
> See inline comments.

Changes incorporated. PTAL.

**Pull request merged**